src/components/Section.js

```diff
@@ -11,7 +11,7 @@ export default class Section {
     });
   }
 
-  addItem(element) {
-    this._containerElement.prepend(element);
+  addItem(newCard) {
```

Comment:
`Section` is a general class. It can render any kind of items, not only cards. Please, rename this variable to `item`

Reply:
So rename addtem to Item?

Reply:
`newCard` to `item`